oci/build-image-manifest.py

```diff
@@ -36,7 +39,7 @@ def hash(data):
         'created': '1970-01-01T00:00:00Z',
         'rootfs': {
             'type': 'layers',
-            'diff_ids': list(map(lambda x: x[1]['digest'], non_empty_layers))
+            'diff_ids': list(map(lambda x: f"sha256:{x[2]}", non_empty_layers))
```

Comment:
I'd prefer to keep the hash algorithm selection and description closer to each other instead of assuming it here by putting the `sha256:` prefix in the `contenthash`.

Reply:
I've renamed the file to `contentsha256` and prefixed it earlier in the code

Reply:
So long as the hash is labelled at the source I think this is ok. 👍.